plugins/services/src/js/components/forms/HealthChecksFormSection.js

```diff
@@ -39,7 +71,9 @@ class HealthChecksFormSection extends Component {
             <FormGroup
               className="column-3"
               showError={Boolean(errors.gracePeriodSeconds)}>
-              <FieldLabel>Grace Period (s)</FieldLabel>
+              <FieldLabel helpText={gracePeriodHelpText}>
```

Comment:
I am not sure I understand what is going on here?
FieldLabel does not have a property `helpText`.
This needs to be added inside the `FormGroup` using a `FieldHelp` component, like so:
```js
<FormGroup
  className="column-3"
  showError={Boolean(errors.gracePeriodSeconds)}>
  <FieldInput
    name={`healthChecks.${key}.gracePeriodSeconds`}
    type="number"
    min="0"
    placeholder="300"
    value={healthCheck.gracePeriodSeconds} />
  <FieldHelp>
    (Optional. Default: 300): Health check failures are ignored within this number of seconds or until the instance becomes healthy for the first time.
  </FieldHelp>
  <FieldError>{errors.gracePeriodSeconds}</FieldError>
</FormGroup>
```

This needs to change for all of them